<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projet CPOOA</string>
  </property>
  <widget class="QWidget" name="pageConnexion">
   <widget class="QGroupBox" name="grpInscrip">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Inscription</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Identifiant :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mot de Passe :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inscripButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Inscription</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inscripIdentifiant">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>35</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="inscripMDP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpConnexion">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>491</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Connexion</string>
    </property>
    <widget class="QLineEdit" name="connexionIdentifiant">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>471</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>35</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="connexionMDP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>471</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="connexionButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Identifiant :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mot de Passe :</string>
     </property>
    </widget>
    <zorder>connexionIdentifiant</zorder>
    <zorder>connexionMDP</zorder>
    <zorder>connexionButton</zorder>
    <zorder>label_2</zorder>
    <zorder>label</zorder>
    <zorder>widget</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
